pages/models.py

```diff
@@ -105,6 +109,11 @@ class RootPage(Page):
             ('image', APIImageChooserBlock(required=False)),
             ('image_alt', blocks.CharBlock(required=False)),
             ('config', blocks.StreamBlock([
+                ('id', blocks.RegexBlock(
```

Comment:
are the options for this going to be documented somewhere?
how do you expect this will be used?

Reply:
the id is the HTML id, which can be used in anchor links to link to a part of the page

i should probably go through and add some more useful help text to all the options. i was also gonna make some persistent pages on `dev` to showcase the different content types